Products/ZenEvents/tests/test_zeneventd.py

```diff
@@ -39,14 +40,14 @@ def setUp(self):
             created_time=1523044529575,
             event_class="/ZenossRM",
             actor=EventActor(
-                element_type_id=10,
+                element_type_id=ModelElementType.DEVICE,
                 element_uuid="575bcf2d-8ca0-47d9-8e63-b4f2c1242ef3",
                 element_identifier="device.loc",
-                element_sub_type_id=100,
+                element_sub_type_id=ModelElementType.COMPONENT,
```

Comment:
`element_sub_type_id` is field of the enumerated type `ModelElementType`.  The value 100 is not a valid enumeration of `ModelElementType`.  This code should have never worked.